### PR TITLE
Add Hyh textstyles

### PR DESCRIPTION
**Pull request by nilsonsasaki**
added a new class with app textstyles, it makes easyier to pick find style

### CODE REVIEW

**Review by yuumasato — APPROVED** [review]

File: lib/constants/styles.dart [hunk 1]
```diff
@@ -15,3 +15,38 @@ const kSettingsTextW700 = TextStyle(fontSize: 16, fontWeight: FontWeight.w700);
...
+
+  //Grayscale Lightest styles
+  static const body14 = TextStyle(fontSize: 14, fontWeight: FontWeight.w400, color: kGrayScaleLightest);
+  static const body14Bold = TextStyle(fontSize: 14, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
+  static const body16 = TextStyle(fontSize: 16, fontWeight: FontWeight.w400, color: kGrayScaleLightest);
+  static const body16Height15 = TextStyle(fontSize: 16, fontWeight: FontWeight.w400, color: kGrayScaleLightest, height: 1.5);
+  static const body16Bold = TextStyle(fontSize: 16, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
+  static const body16Height15Bold = TextStyle(fontSize: 16, fontWeight: FontWeight.w700, color: kGrayScaleLightest, height: 1.5);
+  static const heading18 = TextStyle(fontSize: 18, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
+  static const heading32 = TextStyle(fontSize: 32, fontWeight: FontWeight.w700, color: kGrayScaleLightest);
```
Comment (yuumasato) [inline lib/constants/styles.dart:29]: Is this expected?
The weight is 700, but the style doesn't say bold.

File: lib/constants/styles.dart [hunk 2]
```diff
@@ -15,3 +15,38 @@ const kSettingsTextW700 = TextStyle(fontSize: 16, fontWeight: FontWeight.w700);
...
+
+  //Grayscale Light styles
+  static const heading32Light = TextStyle(fontSize: 32, fontWeight: FontWeight.w700, color: kGrayScaleLight);
```
Comment (yuumasato) [inline lib/constants/styles.dart:32]: Is it expected that `heading32Light` says light but the weight is `w700`?

File: lib/constants/styles.dart [hunk 3]
```diff
@@ -15,3 +15,38 @@ const kSettingsTextW700 = TextStyle(fontSize: 16, fontWeight: FontWeight.w700);
...
+
+  //Grayscale MediumDark styles
+  static const body16BoldMediumDark = TextStyle(fontSize: 16, fontWeight: FontWeight.w700, color: kGrayScaleMediumDark);
+
+  //Grayscale Darkest styles
+  static const body14Darkest = TextStyle(fontSize: 14, fontWeight: FontWeight.w400, color: kGrayScaleDarkest);
+  static const body16Darkest = TextStyle(fontSize: 16, fontWeight: FontWeight.w400, color: kGrayScaleDarkest);
+  static const body16BoldDarkest = TextStyle(fontSize: 16, fontWeight: FontWeight.w700, color: kGrayScaleDarkest);
+  static const heading18Darkest = TextStyle(fontSize: 18, fontWeight: FontWeight.w700, color: kGrayScaleDarkest);
```
Comment (yuumasato) [inline lib/constants/styles.dart:41]: Should the weight be `w400`?

File: lib/constants/styles.dart [hunk 4]
```diff
@@ -15,3 +15,38 @@ const kSettingsTextW700 = TextStyle(fontSize: 16, fontWeight: FontWeight.w700);
...
+
+  //Yellow styles (BrandColor)
+  static const body16BoldYellow = TextStyle(fontSize: 16, fontWeight: FontWeight.w700, color: kYellow);
+  static const heading20Yellow = TextStyle(fontSize: 20, fontWeight: FontWeight.w700, color: kYellow);
+  static const heading24Yellow = TextStyle(fontSize: 24, fontWeight: FontWeight.w700, color: kYellow);
+  static const heading32Yellow = TextStyle(fontSize: 32, fontWeight: FontWeight.w700, color: kYellow);
```
Comment (yuumasato) [inline lib/constants/styles.dart:47]: These are all bold (`w700`), but the naming doesn't reflect that.

**Review by yuumasato — CHANGES_REQUESTED** [review]
The naming and values are a bit inconsistent.

File: lib/constants/styles.dart (hunk 1 above)
Comment (nilsonsasaki) [reply lib/constants/styles.dart:29]: All headings are bold, maybe shold be renamed

File: lib/constants/styles.dart (hunk 2 above)
Comment (nilsonsasaki) [reply lib/constants/styles.dart:32]: i thought just in the color, but maybe should rename to lightGray

**Review by yuumasato — APPROVED** [review]

**Pull request merged**